config/e3sm/machines/config_batch.xml

```diff
@@ -541,30 +541,4 @@
       <queue walltimemax="01:00:00" default="true">lr3</queue>
     </queues>
    </batch_system>
-
-  <batch_jobs>
```

Comment:
These should have reappeared in a workflow XML file for E3SM, right?

Reply:
Yes, it's there now.